x-pack/elastic-agent/pkg/agent/application/application.go

```diff
@@ -73,7 +73,7 @@ func createApplication(
 
 	if configuration.IsStandalone(cfg.Fleet) {
 		log.Info("Agent is managed locally")
-		return newLocal(ctx, log, pathConfigFile, rawConfig, reexec, statusCtrl, uc, agentInfo)
```

Comment:
Can you share a bit more in the PR description on why pathConfigFile contains the wrong info on how this change fixes it?